db/migrate/20221202110535_create_accounts.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAccounts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :accounts do |t|
+      t.string :name, nil: false
+      t.string :slug, nil: false
```

Comment:
A sintaxe tá certa? É `nil: false` mesmo? Acho que na minha época eu usava `null: false` 🤔 😆 

E aqui tbm dá pra colocar igual vc colocou na validação do model:
```suggestion
      t.string :slug, nil: false, unique: true
```

Reply:
Vc sempre vai dar `find` pelo `slug` aqui, né? Então é sempre bom adicionar um `index` nesse campo.

Pode ser que quando vc adiciona `unique: true` ele automaticamente já adiciona um `index` tbm, mas confirma aí... Senão tem que adicionar o index tbm

Reply:
Daí, talvez seja algo como:
```suggestion
      t.string :slug, nil: false, index: { unique: true }
```